recipes/islpy/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "islpy" %}
+{% set version = "2016.2.1" %}
+{% set sha256 = "b23ccca253945ba7b41d091e86b3e0ace6a256238f28abf604de1d06f878895c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - cffi
+  run:
+    - python
+    - cffi
+    - six
+
+test:
+  requires:
+    - pytest
+  imports:
+    - islpy
+
+  commands:
+    - cd $SRC_DIR/test; python -m pytest
```

Comment:
I would use `&&` instead of `; ` to ensure the second happens only when the former is OK. But there is no need to change it now.